k8s/environments/prod/cluster-00/mi/mi-adf-shir.yaml

```diff
@@ -0,0 +1,25 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: mi-adf-shir-deployment
+  namespace: mi
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.chart-image: glob:prod-*
+spec:
+  releaseName: mi-adf-shir
+  chart:
+    git: git@github.com:hmcts/shared-services-flux
+    path: k8s/release/mi/mi-adf-shir
+    ref: master
+  values:
+    labels:
+      app.kubernetes.io/instance: mi-adf-shir-deployment
+      app.kubernetes.io/name: mi-adf-shir-deployment
+    image: sdshmctspublic.azurecr.io/mi/adf-integration-runtime:prod-09b63f3
+    secretsMountPath: ''
+    environment: "prod"
```

Comment:
I'm assuming this is prod only with Prod being hardcoded on the RG etc?

Reply:
Yes, this only runs on prod, if you view the folder hierarchy its part of the prod cluster-00 deployment.